introduction_to_amazon_algorithms/jumpstart_text_classification/Amazon_JumpStart_Text_Classification.ipynb

```diff
@@ -0,0 +1,687 @@
+    "\n",
+    "---"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "eec1725e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "text1 = \"astonishing ... ( frames ) profound ethical and philosophical questions in the form of dazzling pop entertainment\"\n",
+    "text2 = \"simply stupid , irrelevant and deeply , truly , bottomlessly cynical \""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ccb6dd8a",
+   "metadata": {},
+   "source": [
+    "---\n",
+    "Next, we query the finetuned model, parse the response and print the predictions.\n",
+    "\n",
+    "---"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b7b31d7c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "newline, bold, unbold = \"\\n\", \"\\033[1m\", \"\\033[0m\"\n",
+    "\n",
+    "\n",
+    "def query_endpoint(encoded_text):\n",
+    "    response = finetuned_predictor.predict(\n",
+    "        encoded_text, {\"ContentType\": \"application/x-text\", \"Accept\": \"application/json;verbose\"}\n",
+    "    )\n",
+    "    return response\n",
+    "\n",
+    "\n",
+    "def parse_response(query_response):\n",
+    "    model_predictions = json.loads(query_response)\n",
+    "    probabilities, labels, predicted_label = (\n",
+    "        model_predictions[\"probabilities\"],\n",
+    "        model_predictions[\"labels\"],\n",
+    "        model_predictions[\"predicted_label\"],\n",
+    "    )\n",
+    "    return probabilities, labels, predicted_label\n",
+    "\n",
+    "\n",
+    "for text in [text1, text2]:\n",
+    "    query_response = query_endpoint(text.encode(\"utf-8\"))\n",
+    "    probabilities, labels, predicted_label = parse_response(query_response)\n",
+    "    print(\n",
+    "        f\"Inference:{newline}\"\n",
+    "        f\"Input text: '{text}'{newline}\"\n",
+    "        f\"Model prediction: {probabilities}{newline}\"\n",
+    "        f\"Labels: {labels}{newline}\"\n",
+    "        f\"Predicted Label: {bold}{predicted_label}{unbold}{newline}\"\n",
+    "    )"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d1fcc0aa",
+   "metadata": {},
+   "source": [
+    "---\n",
+    "Next, we clean up the deployed endpoint.\n",
+    "\n",
+    "---"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "415912b5",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Delete the SageMaker endpoint and the attached resources\n",
+    "finetuned_predictor.delete_model()\n",
+    "finetuned_predictor.delete_endpoint()"
+   ]
+  }
+ ],
+ "metadata": {
+  "instance_type": "ml.t3.medium",
+  "kernelspec": {
+   "display_name": "Python 3 (Data Science)",
```

Comment:
would this work outside of Studio?

Reply:
Non-blocking. We will do in the next revision.